doc/release-notes.md

```diff
@@ -104,6 +104,10 @@ Updated RPCs
   with the `-json` option set, the following fields: `addresses`, `reqSigs` are no longer
   returned in the tx output of the response. (#20286)
 
+- The `listbanned` RPC now returns two new numeric fields: `ban_duration` and `time_remaining`.
+  Respectively, these new fields indicate the duration of a ban and the time remaining until a ban expires,
+  both in seconds. Additionally, the `ban_created` field is repositioned to come before `banned_until`.
```

Comment:
```suggestion
  both in seconds. Additionally, the `ban_created` field is repositioned to come before `banned_until`.  (#21602)
```

Reply:
done in d3b0b08b0f04d2f1dbebbafd7ab0384dfe045dec